node/src/crypto/hash.rs

```diff
@@ -172,6 +174,90 @@ impl From<Blake2bHash> for Digest {
     }
 }
 
+/// Sentinel hash to be used for hashing options in the case of [None].
+pub const SENTINEL0: Digest = Digest([0u8; 32]);
```

Comment:
```suggestion
pub const SENTINEL0: Digest = Digest([0u8; Digest::LENGTH]);
```
Same for other two sentinels.